render-wasm/src/shapes/fills.rs

```diff
@@ -3,7 +3,10 @@ use skia_safe::{self as skia, Rect};
 use super::Color;
 use crate::uuid::Uuid;
 
-pub const RAW_FILL_DATA_SIZE: usize = 24;
+const MAX_GRADIENT_STOPS: usize = 8;
```

Comment:
Super minor thing: we can leave it as it is and review this later, after we gather more info about the max gradient stops being used, but I would increase it at least to 16